.github/workflows/aws_tests.yaml

```diff
@@ -22,7 +22,8 @@ jobs:
 
       - name: Install python packages & dependencies
         run: |
-          pip install runhouse[aws] pytest tqdm datasets dask
+          pip install tests/test_requirements/additional_requirements.txt
```

Comment:
I don't think we should be installing additional_requirements.txt for these workflows bc several of them aren't being used and are quite large installations (e.g. torch), let's just try including the minimal necessary for the ones here, I think requirements.txt+aws_test_requirements should be sufficient?